### ISSUE TITLE
enum attribute value are not updating

### ISSUE THREAD

**Issue by rashidshkh** [balderdashy/sails-mongo#335]
I have sails js 0.11v and mongodb, when i use .update function of sails js, Except enum attrribute every other value is updating.

Here is my model
module.exports = {
  schema: true,
  attributes: {
  	name: {
	    type: 'string',
	    required: true
	},
	type: {
	    type: 'string',
	    required: true,
	    enum: ['open', 'user']
	},
	
	state: {
	    type: 'integer',
	    enum: [0, 1, 2, 3, 4, 5, 6],
	    defaultsTo : 0
	    
	}
  }
	
};

Below code is responsible for updating values
Challenge.update(
	{id:'55e81f5998e63e7c1ebe4ab6'},
	{state: 2},
	{type: 'user'}
	).exec(function afterwards(err, updated){
	if (err) {
		return res.json(err);
	}
	else{
		return res.json(updated);
	}
});

but it dont update type and state parameter in challenge. It results old data

**Comment by devinivy** [comment]
Odd!  I'll look into it to see if this is an issue in sails-mongo.

**Comment by sailsbot** [comment]
Thanks for posting, @rashidshkh.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
